index.js

```diff
@@ -0,0 +1,76 @@
+exports.push = function(array, element) {
+  var length = array.length;
+  array[length] = element;
+  return array;
+};
+
+exports.pop = function(array) {
+  var length = array.length;
+  if (length) {
+    var item = array[length - 1];
+    array.length = length - 1;
+    return item;
+  } else {
+    return undefined;
+  }
```

Comment:
by default, functions return `undefined`, so strictly speaking not necessary to include `else` block here